subiquitycore/utils.py

```diff
@@ -68,10 +68,18 @@ def is_executable(x):
         for ftype, items in checks[check_type].items():
             for i in items:
                 if not os.path.exists(i):
-                    log.error('FAIL: {} is not found on the'
-                              ' filesystem'.format(i))
-                    env_ok = False
-                    continue
+                    if 'SNAP' in os.environ:
```

Comment:
You still need this in the case of running subiquitycore magic outside a snap (ie. as the package). We should allow that to work properly (and AFAIK this is shared with console-conf...). The intent there is that we shouldn't crash when things like curtin are missing, but rather warn the user early that something is wrong, and that's why the dependencies are explicitly listed and checked (and other places do similar things). The issue is in the snap, you also need to take into account that some pieces may be shipping in the snap, or already be available on a classic system outside snaps (straight on the filesystem). In classic context, non-snap paths should win, and we should only rely on trying the /snap path if is otherwise fails.